solr/contrib/scripting/src/java/org/apache/solr/scripting/response/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Support for transforming via XSL stylesheets search responses.
```

Comment:
The package name suggests nothing of XSLT yet we document it as only for XSLT?  Let's just put all XSLT stuff in the same package? (with or without a .response; I don't think that aspect is important)

Reply:
I think I was captured by the idea of mimicking what was in `/core/` too much..   I like.